src/User/Password/PasswordEncoder.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MsgPhp\User\Password;
+
+/**
+ * @author Roland Franssen <franssen.roland@gmail.com>
+ */
+final class PasswordEncoder implements PasswordEncoderInterface
```

Comment:
`final class PasswordHasher implements PasswordHasherInterface`. [Symfony actually uses the Encoder naming only for legacy purposes.](https://symfony.com/doc/current/security/password_encoding.html)

Imo it would make more sense to rename the "encode" to "hash" part throughout the codebase